demos/android/MASVS-PLATFORM/MASTG-DEMO-0061/MASTG-DEMO-0061.md

```diff
@@ -1,31 +1,41 @@
 ---
 platform: android
-title: Uses of FLAG_SECURE with semgrep
+title: Setting and Clearing FLAG_SECURE in an Activity and Dialog with semgrep
```

Comment:
I am not sure I understand this demo.

Does `context.window` refer to the same window instance as `dialog.window`? So that `dialog.window?.clearFlags` by accident puts `context.window` in danger?

If not, I am not sure about usefulness of this Demo. Isn't it self-explanatory that once you remove the SECURE flag, the window won't be secured?

Reply:
```suggestion
title: Clearing FLAG_SECURE in an Activity and Dialog with semgrep
```